src/main/java/org/jenkinsci/plugins/orgfolder/github/MainLogic.java

```diff
@@ -119,11 +119,11 @@ public void applyOrg(OrganizationFolder of, GitHubSCMNavigator scm) throws IOExc
      */
     public void applyRepo(WorkflowMultiBranchProject item, GitHubSCMNavigator scm) throws IOException {
         if (UPDATING.get().add(item)) {
-            GitHub hub = connect(item, scm);
-            GHRepository repo = hub.getRepository(scm.getRepoOwner() + '/' + item.getName());
-
             BulkChange bc = new BulkChange(item);
             try {
+                GitHub hub = connect(item, scm);
+                GHRepository repo = hub.getRepository(scm.getRepoOwner() + '/' + item.getName());
+
```

Comment:
Not really necessary IIUC—the `BulkChange` has no effect until we start calling `item.setSomething` methods.